Source/mus2mid.h

```diff
@@ -0,0 +1,29 @@
+//
```

Comment:
Is it really necessary to replace the entire `mmus2mid.c` by `mus2mid.c` or is there an isolated change that may be sufficient to apply?